moto/ssm/models.py

```diff
@@ -58,11 +59,74 @@ def describe_response_object(self, decrypt=False):
         return r
 
 
+MAX_TIMEOUT_SECONDS = 3600
+
+
+class Command(BaseModel):
+    def __init__(self, comment='', document_name='', timeout_seconds=MAX_TIMEOUT_SECONDS,
+            instance_ids=[], max_concurrency='', max_errors='',
```

Comment:
We should try to avoid [default mutable arguments](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)

Reply:
Nice! I didn't know about this at all. Great catch. I'll fix this right up. I've learned something new today.